modules/debts_api/lib/debts_api/v0/fsr_form_transform/streamlined_calculator.rb

```diff
@@ -73,7 +73,7 @@ def income_below_gmt_threshold?
         end
 
         def cash_below_gmt_threshold?
-          @asset_data['cashOnHand'] < @gmt_data['gmt_threshold']
+          @asset_data['cashOnHand'].to_f < @gmt_data['gmt_threshold']
```

Comment:
What if @asset_data['cashOnHand'] is nil or blank? Seems like the use of the safe navigation operator here wouldn't be a bad idea.

Additionally, I see a lot of .to_f calls in the code and I wonder if you would benefit from a small helper method here. Something like this, which also provides a default

```
  def fetch_float(key, default = 0.0)
    self.fetch(key, default).to_f
  end
 ```

Reply:
`nil.to_f` and `''.to_f` both return 0.0 which is the expected behavior.